tests/integration/17.SecurityTag.Tests.ps1

```diff
@@ -28,20 +28,38 @@ Describe -Name "SecurityTagAssignment" -Tag "Get" -Fixture {
         #We load the mod and establish connection to NSX Manager here.
 
         #Put any setup tasks in here that are required to perform your tests.  Typical defaults:
+        Write-Verbose -Verbose "Performing setup tasks for SecurityTag tests"
         Import-Module $pnsxmodule
         $script:DefaultNsxConnection = Connect-NsxServer -vCenterServer $PNSXTestVC -NsxServerHint $PNSXTestNSX -Credential $PNSXTestDefViCred -ViWarningAction "Ignore"
 
+        ## GUID to use as suffix for some temporary vSphere/NSX objects, so as to prevent any naming conflict with existing objects
+        $strSuffixGuid = [System.Guid]::NewGuid().Guid.Replace("-","")
```

Comment:
Only use powernsx_ (like other test suite?)

Reply:
Howdy-  We could, if we want to leave a chance for object naming collision (although, granted, it would still be fairly improbable that a collision would happen even with just the prefix).  I added the GUID suffix to drive the possibility of naming collision closer to zero.  If a lower chance of collision is better, we could update other test suites at some point to also employ a GUID suffix.